### PR TITLE
More consistent haptics on web overlays and entities

### PR DESCRIPTION
**Pull request by hyperlogic**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 6738c0eadebaedf925f78593e1d2919db4bd6df3 available at: https://gist.github.com/10e8ae332ac93b86395777bcde53dec3<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9607-6738c0e.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9607-6738c0e.dmg

**Comment by hyperlogic** [comment]
QA test plan, HMD mode (vive or oculus)
    * Verify that the left and right stylus triggers a haptic pulse when the tip of the pen touches the surface of the tablet.
   * Verify that the hand lasers trigger a haptic pulse when the trigger is pulled on the surface of the tablet.
   * Verify that the tablet "home" button triggers a haptic pulse when touched with a stylus.
   * Verify that the tablet "home" button triggers a haptic pulse when the hand laser is over the button and the primary trigger is pressed.
   * Create a web entity.
   * Verify that the left and right stylus appear and trigger a haptic pulse when the tip of the pen touches the surface of the web entity.
   * Verify that the hand lasers trigger a haptic pulse when the trigger is pulled on the surface of the tablet.
   * Verify that there are no cases when the haptic pulses buzz because they are triggered repeatedly

**Comment by claudioshg** [comment]
Testing

File: scripts/system/controllers/handControllerGrab.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ Script.include("/~/system/libraries/controllers.js");
 // add lines where the hand ray picking is happening
 //
 var WANT_DEBUG = false;
-var WANT_DEBUG_STATE = false;
+var WANT_DEBUG_STATE = true;
```
Comment (sethalves) [inline scripts/system/controllers/handControllerGrab.js]: could you flip this back to false?

**Comment by claudioshg** [comment]
### Desktop PC - Windows 10 Home - CPU 4.0GHz Intel i7 - GTX1070 - RAM 16GB - Tested with Oculus Rift + Oculus Touch

**PASS**
Tested Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9607-6738c0e.exe


- Verify that the left and right stylus triggers a haptic pulse when the tip of the pen touches the surface of the tablet. PASS
- Verify that the hand lasers trigger a haptic pulse when the trigger is pulled on the surface of the tablet. PASS
- Verify that the tablet "home" button triggers a haptic pulse when touched with a stylus. PASS
- Verify that the tablet "home" button triggers a haptic pulse when the hand laser is over the button and the primary trigger is pressed. PASS
- Create a web entity. PASS
- Verify that the left and right stylus appear and trigger a haptic pulse when the tip of the pen touches the surface of the web entity. PASS
- Verify that the hand lasers trigger a haptic pulse when the trigger is pulled on the surface of the tablet. PASS
- Verify that there are no cases when the haptic pulses buzz because they are triggered repeatedly. PASS

**NOTE**: No problems whatsoever. Everything worked as it should.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 244437d971c9fc2a92da662497daec39ab17f23c available at: https://gist.github.com/10e8ae332ac93b86395777bcde53dec3<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9607-244437d.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9607-244437d.dmg

**Pull request merged**